api/controllers/linux/install/hostpreflight.go

```diff
@@ -87,6 +93,20 @@ func (c *InstallController) RunHostPreflights(ctx context.Context, opts RunHostP
 	return nil
 }
 
+func (c *InstallController) getStateFromPreflightsOutput(ctx context.Context) statemachine.State {
+	output, err := c.GetHostPreflightOutput(ctx)
+	// If there was an error getting the state we assume preflight execution failed
+	if err != nil {
+		c.logger.WithError(err).Error("error getting preflight output")
+		return StatePreflightsExecutionFailed
+	}
+	// If there is no output, we assume preflights succeeded
```

Comment:
is that a correct assumption though?

Reply:
Ah thank you I wanted to raise that and missed it in my review. I was initially failing, but then I started thinking that the absence of output shouldn't necessarily mean it failed?